### PR TITLE
27.0.0

### PR DESCRIPTION
**Pull request by blizzz**
* [x] move to releases folder once the time is right
* [x] update path to archive

### CODE REVIEW

**Comment by blizzz** [comment]
Tests as expected :+1:

File: config/config.php [hunk 1]
```diff
@@ -108,6 +108,38 @@
...
+6N7YkvKlPMCSk3ttIgIH04J+CMYZWzW0UVRn2NV2ID8e22+tiskWNVK6uvTszIX5
+AU45erykEtmF8RqZR5wTPBOc4kjrryTnKhuNsEPKtEjQlxpI5/PdV5fE+eegTGeh
+EGDLJyXxKje9pcfLoid0xQ==',
+			],
+		],
+		'26.0.2' => [
+			'30' => [
+				'latest' => '27.0.0',
+				'internalVersion' => '27.0.0.8',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-27.0.0.zip',
```
Comment (AndyScherzinger) [inline config/config.php:131]: same here

File: config/config.php [hunk 2]
```diff
@@ -635,34 +667,34 @@
 	'beta' => [
 		'27' => [
 			'100' => [
-				'latest' => '27.0.0 RC4',
-				'internalVersion' => '27.0.0.7',
-				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-27.0.0rc4.zip',
+				'latest' => '27.0.0',
+				'internalVersion' => '27.0.0.8',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-27.0.0.zip',
```
Comment (AndyScherzinger) [inline config/config.php:672]: and here

File: config/config.php [hunk 3]
```diff
@@ -635,34 +667,34 @@
...
 			],
 		],
 		'26.0.2' => [
 			'100' => [
-				'latest' => '27.0.0 RC4',
-				'internalVersion' => '27.0.0.7',
-				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-27.0.0rc4.zip',
+				'latest' => '27.0.0',
+				'internalVersion' => '27.0.0.8',
+				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-27.0.0.zip',
```
Comment (AndyScherzinger) [inline config/config.php:688]: and here

File: tests/integration/features/beta.feature [hunk 4]
```diff
@@ -712,16 +712,16 @@ Feature: Testing the update scenario of beta releases
     And the installation mtime is "11"
     When The request is sent
     Then The response is non-empty
-    And Update to version "27.0.0.7" is available
-    And URL to download is "https://download.nextcloud.com/server/prereleases/nextcloud-27.0.0rc4.zip"
+    And Update to version "27.0.0.8" is available
+    And URL to download is "https://download.nextcloud.com/server/prereleases/nextcloud-27.0.0.zip"
```
Comment (AndyScherzinger) [inline tests/integration/features/beta.feature:716]: :)

**Comment by AndyScherzinger** [comment]
Not sure I read it correctly as I would have expected the target folder to be releases not prereleases but maybe that is reflected in todo 1 ?

**Review by AndyScherzinger — APPROVED** [review]

**Comment by blizzz** [comment]
> Not sure I read it correctly as I would have expected the target folder to be releases not prereleases but maybe that is reflected in todo 1 ?

Exactly. That's also why this PR is a draft.

**Pull request merged**